C/guide/ch_txns.xml

```diff
@@ -82,42 +82,27 @@
       <title>Choosing a Register Style</title>
 
       <para>&app; offers several options for viewing your registers. The default style is <guilabel>Basic
-        Ledger</guilabel> mode, which displays only the summary of splits affecting the current
+        Ledger</guilabel>, which displays only the total of splits affecting the current
```

Comment:
It's only a total if we are dealing with a 1-to-many split, even there it's the 1 you are seeing. Or this could be a many-to-many split, in which case it's certainly not a total, it's just a split amount. So I think it's better to simply say: 
`which displays only the amount of the split affecting the current ledger.`
I know there is more to it with the transfer account being shown (or "split transaction" if there are more than 2 splits) but that's just more info than needed here I think.

Reply:
Also maybe we should say register instead of ledger. It seems the 2 are used but I see register used below as well.